circle.yml

```diff
@@ -3,18 +3,18 @@ machine:
     version: oraclejdk7
   environment:
     TERM: dumb
-    GRADLE_OPTS: '-Dorg.gradle.jvmargs="-Xmx3400m -XX:+HeapDumpOnOutOfMemoryError"'
+    GRADLE_OPTS: '-Dorg.gradle.jvmargs="-Xmx3g -XX:+HeapDumpOnOutOfMemoryError"'
 
 dependencies:
   override:
     - ./gradlew resolveConfigurations
 
 test:
   override:
-    - ./gradlew --daemon --parallel build testBouncyCastle testSun
+    - ./gradlew build testBouncyCastle testSun
```

Comment:
I don't think so.

On Thu, Jul 21, 2016 at 12:03 PM Mark Elliot <notifications@github.com>
wrote:

> In circle.yml
> <https://github.com/palantir/http-remoting/pull/174#discussion_r71766957>:
>
> >
> >  dependencies:
> >    override:
> >      - ./gradlew resolveConfigurations
> >
> >  test:
> >    override:
> > -    - ./gradlew --daemon --parallel build testBouncyCastle testSun
> > +    - ./gradlew build testBouncyCastle testSun
>
> does testBouncyCastle still fork a docker container?
>
> —
> You are receiving this because you modified the open/close state.
> Reply to this email directly, view it on GitHub
> <https://github.com/palantir/http-remoting/pull/174/files/78135c55be6b985b7b674f7eba010c45224ede13#r71766957>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AGOdwaPPdeLRsVwo1A5BXjA6QbYjTIJfks5qX8JlgaJpZM4JSCWu>
> .
>